progress_observer/progress_observer.go

```diff
@@ -121,6 +126,8 @@ func ObserveIndexers(ctx context.Context, sourceUrl, targetUrl string, m *metric
 	m.RecordCount(len(matches), sourceName, targetName, metrics.MatchCount)
 	m.RecordCount(len(unknwonByTarget), sourceName, targetName, metrics.UnknownCount)
 	m.RecordCount(len(unknownBySource), targetName, sourceName, metrics.UnknownCount)
+	m.RecordCount(totalSourceLag, sourceName, "", metrics.IngestLag)
+	m.RecordCount(totalTargetLag, targetName, "", metrics.IngestLag)
```

Comment:
record count would skip adding tags if value is empty right? Otherwise we will get tags with empty values

Reply:
Yes, it won't add empty tags at all: https://github.com/ipni/index-observer/blob/main/progress_observer/metrics/metrics.go#L111